tiled_layers/tile-cache/README.md

```diff
@@ -1,30 +1,39 @@
-# Tile cache
+# ArcGIS tiled layer (tile cache)
 
-Create a basemap from a local tile cache.
+Load an offline copy of a tiled map service as a basemap.
 
-![](TileCache.png)
+![Image of ArcGIS tiled layer tile cache](TileCache.png)
```

Comment:
technically this alt text could remain as it is, or we can keep it to match the title: not fussed either way
```suggestion
![Image of tile cache](TileCache.png)
```